elements/rocky-container-stackhpc/containerfiles/9-stackhpc

```diff
@@ -0,0 +1,16 @@
+FROM docker.io/rockylinux/rockylinux:9
+
+RUN dnf group install -y 'Minimal Install' --allowerasing && \
+    dnf install -y findutils util-linux \
+    https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/Packages/cloud-init-23.1.1-12.el9.noarch.rpm
```

Comment:
Do we know how long CS9 keeps packages in their repos for?

Reply:
They are keeping 5 versions of each package, hopefully the updated version lands in RHEL 9.3 and we can stop doing this.